src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -63,7 +65,25 @@ export class IssuesDataTable extends DataSource<Issue> implements FilterableSour
       .pipe(
         // maps each change in display value to new issue ordering or filtering
         map(() => {
+          // check if the current group is in the filter
+          const groupFilterAsGithubUser = this.filter.assignees.map((selectedAssignee) => {
+            return GithubUser.fromUsername(selectedAssignee);
+          });
+          const groupFilterAsMilestone = this.filter.milestones.map((selectedMilestone) => {
+            return Milestone.fromTitle(selectedMilestone);
+          });
+
+          const isGroupInFilter =
+            groupFilterAsGithubUser.filter((githubUser) => this.group?.equals(githubUser)).length !== 0 ||
+            groupFilterAsMilestone.filter((milestone) => this.group?.equals(milestone)).length !== 0;
```

Comment:
```suggestion
            groupFilterAsGithubUser.some((githubUser) => this.group?.equals(githubUser)) ||
            groupFilterAsMilestone.some((milestone) => this.group?.equals(milestone));
```
This is not very efficient, `Array.some()` is built for this.